README.md

```diff
@@ -12,22 +11,23 @@ Seriously, [check it out](http://www.trello.com/).
 
 [Full API documentation](http://www.rubydoc.info/gems/ruby-trello).
 
-## Installation
-
-```
-# gem install ruby-trello
-```
-
 Full Disclosure: This library is mostly complete, if you do find anything missing or not functioning as you expect it
-to, please [let us know](https://trello.com/card/spot-a-bug-report-it/4f092b2ee23cb6fe6d1aaabd/17).
+to, please just create an issue.
 
-Supports Ruby 2.1.0 or newer.
+## Requirements
 
+Use the newest version for Ruby 2.1.0 or newer support.
```

Comment:
I'd like to propose that we only support Ruby 2.5 or newer. Ruby 2.1.0 has reached its EOL for quite a while.

Reply:
I like this. I have changed them in #278. And what version of Rails do you think we should support?

Reply:
I think we should support 5.2 and 6.x

Reply:
👌